<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>1024</width>
      <height>657</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/pic/photo_2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuZwj">
    <property name="title">
     <string>选项(&amp;S)</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_D"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_M"/>
   </widget>
   <addaction name="menuZwj"/>
   <addaction name="menu"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="action_3"/>
   <addaction name="action_D"/>
   <addaction name="action_4"/>
   <addaction name="action_7"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_3">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/pic/images/注册.PNG</normaloff>:/new/pic/images/注册.PNG</iconset>
   </property>
   <property name="text">
    <string>注册(&amp;Z)</string>
   </property>
  </action>
  <action name="action_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/pic/images/修改.PNG</normaloff>:/new/pic/images/修改.PNG</iconset>
   </property>
   <property name="text">
    <string>修改密码(&amp;X)</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>退出(&amp;T)</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/pic/images/通信设置.PNG</normaloff>:/new/pic/images/通信设置.PNG</iconset>
   </property>
   <property name="text">
    <string>通信设置（&amp;S）</string>
   </property>
  </action>
  <action name="action_M">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/pic/images/帮助.PNG</normaloff>:/new/pic/images/帮助.PNG</iconset>
   </property>
   <property name="text">
    <string>帮助(&amp;M)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/pic/images/登录.png</normaloff>:/new/pic/images/登录.png</iconset>
   </property>
   <property name="text">
    <string>登录(&amp;D)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
